framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
Well. It *is* broken currently. Command disabling *must not* control the enabled state of buttons that inherited their state from their parent chain.

I personally expect a disabled button to *not* react on anything. Neither clicks nor keyboard shortcuts. But this is the case in the moment.